kratos/containers/model.h

```diff
@@ -57,6 +56,7 @@ namespace Kratos
 * @author Riccardo Rossi
 */
 class KRATOS_API(KRATOS_CORE) Model final
+    : public DataValueContainer
```

Comment:
Model is a very basic class. Our (@KratosMultiphysics/technical-committee ) proposal would be to have the model to "have" a data_value_container instead of "being" a datavalue container.

this way one would use it on the line of 

~~~cpp
current_model.GetDataValueContainer()....
~~~

Reply:
Take a look now, @RiccardoRossi